week6_outro/bandits.ipynb

```diff
@@ -1,5 +1,20 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run this if in COLAB\n",
+    "!pip install --upgrade https://github.com/Theano/Theano/archive/master.zip\n",
```

Comment:
This was intended for the BNN assignment in the YSDA course. Could you remove it from the Coursera version?